exporter/coralogixexporter/factory.go

```diff
@@ -46,5 +45,18 @@ func createDefaultConfig() config.Exporter {
 }
 
 func createTraceExporter(_ context.Context, set component.ExporterCreateSettings, config config.Exporter) (component.TracesExporter, error) {
-	return nil, fmt.Errorf("Coralogix exporter is not implemented yet")
+	cfg := config.(*Config)
+	err := cfg.Validate()
```

Comment:
Validate is called automatically by the collector itself before your exporter is created.

Reply:
Thanks, I will remove this code line